src/styles/reset.scss

```diff
@@ -0,0 +1,42 @@
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  background-repeat: no-repeat;
+}
+
+::before,
+::after {
```

Comment:
Are you sure that all these resets are useful here?

Reply:
.There is a large reset file, I partially delete it for small projects. For some reason, I missed these CSS rules here.